src/main/java/competition/Robot.java

```diff
@@ -45,6 +45,8 @@ public BaseRobotComponent getInjectorComponent() {
 
     @Override
     public void simulationInit() {
+        // teleport the robot to the field, bottom left ish pointing towards the middle
```

Comment:
worth noting that "pointing towards the middle" must be coming from somewhere else (probably something initialized to 0 degrees)

Reply:
Yeah that's right, I hadn't messed with orientation much and this comment was more aspirational than what's happening, hah, I'll add it or remove that part of the comment in the shorturn